tutorial/katacoda-scenarios/flyte-install/index.json

```diff
@@ -19,16 +17,17 @@
       "text": "finish.md"
     },
     "intro": {
-
+      "code": "foreground.sh",
       "text": "intro.md"
     }
   },
   "environment": {
+    "hideintro": true,
     "uilayout": "terminal-terminal",
     "showdashboard": true,
-    "dashboards": [{"name": "Flyte Get Started", "href": "https://flyte.org"},{"name": "Flyte Console", "href": "https://[[HOST_SUBDOMAIN]]-80-[[KATACODA_HOST]].environments.katacoda.com"}]
+    "dashboards": [{"name": "Flyte Get Started", "href": "https://flyte.org"},{"name": "Flyte Console", "href": "https://[[HOST_SUBDOMAIN]]-30081-[[KATACODA_HOST]].environments.katacoda.com"}]
   },
   "backend": {
-    "imageid": "kubernetes-cluster-running:1.18"
+    "imageid": "ubuntu:2004"
```

Comment:
Can we use a smaller image?

Reply:
I will rewrite the entire tutorial, for now, I can merge.